admin_pages/transactions/EE_Admin_Transactions_List_Table.class.php

```diff
@@ -181,8 +169,8 @@ public function column_id(EE_Transaction $transaction)
         );
         $content = '
         <span class="ee-entity-id">
-            <a  class="ee-aria-tooltip" 
-                href="' . $view_lnk_url . '" 
+            <a  class="ee-aria-tooltip"
```

Comment:
A extra space can be removed between `<a` and `class`. There are some more instances in the code such line 247.